File: Math-Packs/r/r-basics/task6.R
task6<-function(){
  ## your code here
  x=runif(1000,-1,1);
  y=rnorm(1000);
  z=rnorm(1000,0,10);
  t=rnorm(1000,0.01,1);
  print(t.test(x,y));
  print(t.test(x,z));
  print(t.test(x,t));
  print(t.test(y,z));
  print(t.test(y,t));
  print(t.test(z,t));
}

File: Math-Packs/octave/octave-indexing/task6.m
function C1 = task6 (C)
  C(mod(C, 2) == 1)*=2;
  C1 = C;
endfunction


#############################################
##                Test code                ##
##     Do not delete it or correct it!     ##
#############################################
%!test                                     ##
%!  C = randi ([-50, 50], 1, 25)           ##
%!  C1 = task6 (C)                         ##
%!test                                     ##
%!  D = randi ([-50, 50], 5, 15)           ##
%!  D1 = task6 (C)                         ##
%!test                                     ##
%!  E = []                                 ##
%!  E1 = task6 (E)                         ##
%!test                                     ##
%!  F = randi ([-100, 100], 10)            ##
%!  F1 = task6 (F)                         ##
#############################################